vscode/microsoft-kiota/src/commands/Command.ts

```diff
@@ -0,0 +1,5 @@
+export abstract class Command {
+  public abstract toString(): string;
```

Comment:
Struggling to understand the usecase of toString method

Reply:
It's going to change to `getName` . It returns the name of the command